help/en/releasenotes/current-draft-note.shtml

```diff
@@ -21,8 +21,8 @@
 
         <h4>CBUS</h4>
             <ul>
+                <li>CANPiWi v1 Firmware - No need for Node Manager to request Number of Events.</li>
                 <li>Fixed Internal Sensor Manager set as default, not CBUS Sensor Manager, affecting 4.19.6 onwards.</li>
-                <li>Simulated Connections - Output Interval option added to Additional Connection Settings.</li>
```

Comment:
Is it intended that this line is removed?

Reply:
Yes to be removed,
line carry-over from https://www.jmri.org/releasenotes/jmri4.23.3.shtml